CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ### 0.3.5 (Next)
 
-* Your contribution here.
```

Comment:
Could you bring this back to the CHANGELOG? This way new contributors will be able to easily see that a new entry should be added here.

Reply:
my bad!